config/settings.yml

```diff
@@ -932,6 +932,7 @@
   :level_vcloud: info
   :level_vim: warn
   :level_remote_console: info
+  :level_azure_stack: info
```

Comment:
alphabetize please? (I'm actually) surprised the remote console isn't already alphabetized)